api/models/vpp/interfaces/keys.go

```diff
@@ -33,6 +33,12 @@ var (
 		Version: "v2",
 		Type:    "interfaces",
 	})
+
+	ModelSpan = models.Register(&Span{}, models.Spec{
+		Module:  ModuleName,
+		Version: "v2",
+		Type:    "span",
+	}, models.WithNameTemplate("{{.InterfaceFrom}}/to/{{.InterfaceTo}}"))
```

Comment:
What about having two spans with same _from_ and _to_ interfaces, but different _state_or _isL2_. Is that supported? Because if so, it will make a collision in the keys.

Reply:
Let's keep it like this for now. If we run into some cases we can come back to this.

Reply:
I've also thought about this and here what I've found:
> If SPAN is configure on an interface with both L2 and without L2, you will get packet mirrored twice [...].  If interface is in L3 mode, L2 SPAN will not have any effect.   

https://lists.fd.io/g/vpp-dev/topic/span_questions/18098046

Reply:
as I understood if you configure both with L2 and without it will either mirror twice or not have any effect. It looks like useless to support this